### ISSUE TITLE
Could not find module `ember-copy` imported from `ember-autoresize/system/font-loaded`

### ISSUE THREAD

**Issue by lougreenwood** [tim-evans/ember-autoresize#104]
Hi, I get this error after upgrading to 1.3.1:

```
Could not find module `ember-copy` imported from `ember-autoresize/system/font-loaded`
```

I see `ember-copy` in the package.json for the add-on, but not in my project's `node_modules`.

Seems that if I add `ember-copy` to my project dependencies it fixes the problem?

```
ember-cli: 3.3.0
node: 10.4.1
os: darwin x64
```

**Comment by tim-evans** [comment]
@lougreenwood I need to publish, there was a bad deploy, sorry!

**Comment by tim-evans** [comment]
Should be fixed by 1.3.2
